helm/fluent-logshipping-app/templates/priorityClass.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.fluentbit.customEnabled }}
+apiVersion: scheduling.k8s.io/v1
+description: High-priority class for fluentbit pods only.
+kind: PriorityClass
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ .Values.fluentbit.priorityClass.name }}
+preemptionPolicy: PreemptLowerPriority
+value: 200000000
```

Comment:
Also, maybe we can override this one so it should be part of the values?

Reply:
Sure.